packages/trader/src/App/Containers/ProgressSliderStream/__tests__/progress-slider-stream.spec.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { mockStore } from '@deriv/stores';
+import TraderProviders from '../../../../trader-providers';
+import ProgressSliderStream from '../progress-slider-stream';
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    ProgressSlider: () => <div>Mocked Progress Slider</div>,
+}));
+
+const contract_info = {
+    contract_type: 'TEST',
+    date_expiry: 1222222224,
+    date_start: 1222222222,
+    tick_count: 2,
+    tick_stream: [
+        { epoch: 1, tick: 1, tick_display_value: '300' },
+        { epoch: 2, tick: 2, tick_display_value: '302' },
+    ],
+};
+
+describe('<ProgressSliderStream />', () => {
+    it('should not render <ProgressSliderStream /> if contract_info is falsy', () => {
+        const mock_root_store = mockStore({});
```

Comment:
I used mockStore in each test case regarding to this discussion: https://github.com/binary-com/deriv-app/pull/9155#discussion_r1244585341